frontend/src/pages/CreateDropPage.js

```diff
@@ -0,0 +1,234 @@
+			return;
+		}
+		if (formattedUriList.length === 0) {
+			setCreateDropResult("No NFT URIs entered.");
+		}
+
+		setCreateDropResult("Sending request to the blockchain ...");
+		setCreateDropData({
+			formattedDropTime: formattedDropTime/1000,
+			formattedUriList: formattedUriList,
+			formattedNftPrice: formattedNftPrice,
+			newNftName: newNftName,
+			newNftTokenName: newNftTokenName,
+			unformattedNftPrice: newNftPrice,
+			unformattedDropTime: newDropTimeDate
+		});
+		setConfirmDropCreateDialogIsOpen(true);
+	};
+
+	return <CenterFlexBoxMedium>
+		<HeaderTypography style={{ fontSize: "26px", paddingBottom: "16px" }}>Create new drop</HeaderTypography>
+
+		<ParagraphTypography style={{ paddingBottom: "24px" }}>
+			Enter the required information below, press continue, confirm the entered data and pay the required fee in MetaMask to create a new drop.
+		</ParagraphTypography>
+
+		<Box component="form" onSubmit={handleCreateNewDrop} noValidate sx={{ mt: 1 }}>
+			<Grid container direction="column" justifyContent="left" alignContent="left">
+				<ParagraphTypography>Enter the name of every new NFT in the drop. </ParagraphTypography>
+				<TextField sx = {{ ...inputFieldStyle, paddingBottom: "20px"  }} margin="normal" required fullWidth id="newNftName" label="NFT name" name="newNftName" autoFocus />
+
+				<ParagraphTypography>Enter NFT token name of every new NFT in the drop. Must be at most 11 characters long.</ParagraphTypography>
+				<TextField sx = {{ ...inputFieldStyle, paddingBottom: "20px"  }} margin="normal" required fullWidth id="newNftTokenName" label="NFT token name" name="newNftTokenName" autoFocus />
+
+				<ParagraphTypography>Enter the price of every new NFT in the drop in ETH. </ParagraphTypography>
+				<TextField sx = {{ ...inputFieldStyle, paddingBottom: "20px"  }} margin="normal" required fullWidth id="newNftPrice" label="New NFT price (ETH)" name="newNftPrice" autoFocus />
+
+				<ParagraphTypography style={{ paddingBottom: "20px" }}>Select the time at which the drop will be dropped. </ParagraphTypography>
+				<LocalizationProvider dateAdapter={AdapterMoment} style={{ paddingBottom: "20px" }}>
+					<DateTimePicker
+						ampm={false}
+						margin="normal"
+						renderInput={(props) => <TextField id="asd" {...props} sx={{ ...inputFieldStyle, color: "white" }}/>}
+						label="Drop date & time"
+						value={newDropTimeDate}
+						onChange={(newValue) => {
+							setNewDropTimeDate(newValue);
+						}}
+					/>
+				</LocalizationProvider>
+
+				<ParagraphTypography style={{ paddingTop: "20px" }}>Enter the URIs of the new NFTs. Enter exactly one URI per line. The number of URIs determines the number of NFTs in the drop. </ParagraphTypography>
+				<TextField multiline rows={20} sx = {{ ...inputFieldStyle, paddingBottom: "20px"  }} margin="normal" required fullWidth id="nftUriList" label="NFT image URIs" name="nftUriList" autoFocus />
```

Comment:
is this text field `NFT image URIs` required? I could create a drop although it was left empty. The Asterisk symbol is displayed in the UI

Reply:
Good catch. I added a condition in the formatting function to catch this.